packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -106,11 +112,14 @@ test.describe('Chat Composite custom data model', () => {
   });
 
   test('can be viewed by user[1]', async ({ pages }) => {
-    const testMessageText = 'How the turn tables';
     const page = pages[0];
     await page.bringToFront();
-    await sendMessage(page, testMessageText);
-    await waitForMessageDelivered(page);
+    await page.type(dataUiId(IDS.sendboxTextField), 'How the turn tables');
```

Comment:
these changes look like needing reverted

Reply:
Actually, only partially. I need to merge.